src/Compilers/VisualBasic/Portable/VisualBasicParseOptions.vb

```diff
@@ -359,5 +359,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Overrides Function GetHashCode() As Integer
             Return Hash.Combine(MyBase.GetHashCodeHelper(), CInt(Me.SpecifiedLanguageVersion))
         End Function
```

Comment:
>End Function [](start = 8, length = 12)

It looks like the only change in this file is an addition of an empty line after this one. Consider reverting changes to this file.